### ISSUE TITLE
Still not getting all notifications

### ISSUE THREAD

**Issue by RoyBatty64** [Nexus-Mods/web-issues#507]

**Comment by RoyBatty64** [comment]
I'm still missing random notifications on my tracked mods and own mods, this is extremely frustrating as an author. You seem to have some issue with your CDN being synced, please fix this.

I can't give you examples because it's completely random if it happens or not or which mod it happens on.

**Comment by iamtiz** [comment]
Did you notice a pattern, for example specific mods, games, or type of notifications?

**Comment by EvilOssie** [comment]
Verifying the issue - I've missed a few notices about Willow specifically, and given that I'm part of llamaRCA's volunteer support crew, it's a bit annoying. No idea if there are other notices I've missed since, well...

**Comment by CharissaM** [comment]
I'm also still not getting all notifications. One thing I've noticed is that it's especially prevalent when I've tracked mod authors instead of individual mods. One example: I author tracked [alpinehunter](https://www.nexusmods.com/skyrim/users/50651486) after they released Summer Overhaul, then when they released [Unbelievable Grass 2 Redux](https://www.nexusmods.com/skyrim/mods/88876) a couple days later I didn't get a notification. Oddly, I did get a notification of them posting [this image](https://www.nexusmods.com/skyrim/images/690540). I've noticed several other cases of me tracking an author and not getting mod release and update notifications. 

Anyway, despite the specific example above, some individual mods I track also don't seem to deliver notifications. I mostly notice this when I get a notification for an update and then see there was an update before that which I wasn't notified of. 

I have also noticed that reply notifications to comments I left on mod pages are spotty. 

I mostly track mods and authors for Skyrim and its Special Edition.
